---
layout: post
title:  "Useful Tutorials"
date:   2016-05-10 00:12:00
categories: CyVerse Tutorials
---

A compilation of various tutorials that you will need to get started from account setup to app installation onto agave.

#**Account Setup**

Before you start using any of the tools available through CyVerse you must first set up accounts with CyVerse, TACC(Texas Advanced Computing Center), XSEDE(Extreme Science and Engineering Discovery Environment) as well as requesting allocations for access to the Agave API.       [Account Setup for CyVerse](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/Account-setup.md). Please note that while Putty is free, CyberDuck is not.

#**Validate on Atmosphere**

Short and quick guide and videos pertaining to [getting familiar with Atmosphere](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/Validate%20on%20Atmosphere.md).

#**Stampede Quick Start**

A guide for users who are unfamiliar with the basics of [the Stampede supercomputer](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/Stampede-guide.md) or only ave a cursory knowledge of the system.

#**Installing Functions and Sims**

While not terribly difficult to do and even though Atmosphere and Stampede provide many options for your GWAS analysis tools you can also install your own methods for your personal use.
 
[Installing Functions](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/Your%20functions.md)

[Installing Sims](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/Your%20sims.md)

#**Validate WorkFlow Tutorial**

This section will get you up to speed on the validate workflow and all steps included. 

[**Simulate**](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/Simulate.md) is the first step of the workflow process and is used to create sample data to run through the GWAS tools. 

[**FaST-LMM**](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/FaST-LMM%20Docs.md) the main genome wide association studies tool that we have used. FaST-LMM stands for Factored Spectrally Transformed Linear Mixed Models. 

[**GEMMA**](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/GEMMA%20Doc.md) is the software implementing the Genome-wide Efficient Mixed Model Association algorithm for a standard linear mixed model and some of it's close relatives for Genome-Wide Association Studies.

[**Qxpak**](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/Qxpak.md) is a package for versatile statistical genomics, specifically designed for sophisticated quantitative trait loci and association analyses. 

[**PLINK**](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/PLINK.md) is useful for handling both linear and logistic regression models and for file format conversion. It's a popular bioinformatics tool designed, for basic, large-scale analyses in a computationally efficient manner.   

[**Winnow**](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/Winnow.md) is a Python-compatible known truth testing tool for genome wide association studies used to compare the data from simulate.

[**Demonstrate**](https://github.com/UNCW-iPlant/Quickstart-guide/blob/master/docs/Demonstrate.md) is the final step in the Validate process known-truth pipeline for CyVerse.
